<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2025-01-15 21:49:11.842 ***</b>
=ERROR REPORT==== 15-Jan-2025::21:49:11.841826 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.247154.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1975}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1936}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2025-01-15 21:49:11.842 ***</b>
=CRASH REPORT==== 15-Jan-2025::21:49:11.842114 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.247159.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2428)
    ancestors: [<0.247154.0>]
    message_queue_len: 0
    messages: []
    links: [<0.247154.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 987
    stack_size: 29
    reductions: 594
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2025-01-15 21:49:52.207 ***</b>
=ERROR REPORT==== 15-Jan-2025::21:49:52.207461 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.251503.0&gt;,
                               {#Ref&lt;0.4100474249.3205234689.77465&gt;,1003909,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.4100474249.3205234689.77465&gt;,1003909,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2025-01-15 21:49:52.207 ***</b>
=CRASH REPORT==== 15-Jan-2025::21:49:52.207776 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.251523.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.4100474249.3205234689.77465&gt;,1003909,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [&lt;0.251503.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 414
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2025-01-15 21:54:35.020 ***</b>
=CRASH REPORT==== 15-Jan-2025::21:54:35.020352 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.257335.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.4100474249.3206545409.215728&gt;,9540,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1696)
    ancestors: [&lt;0.257333.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28147
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-15 21:54:44.584 ***</b>
=ERROR REPORT==== 15-Jan-2025::21:54:44.583887 ===
** Generic server <0.258627.0> terminating 
** Last message in was {nodedown,'peer_down_crash-370244-11901@24acc0d94d0f'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-370244-11901",
                              peer_down => crash},
                            'peer_down_crash-370244-11901@24acc0d94d0f',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,50,47,98,105,110,
                             47,101,114,108],
                            ["-sname","peer_down_crash-370244-11901",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQDI0YWNjMGQ5NGQwZgAD8kMAAAAAZ4gcow=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-370244-11901@24acc0d94d0f'}},
                            {<0.258601.0>,
                             #Ref<0.4100474249.3199205379.178544>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-370244-11901@24acc0d94d0f'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2025-01-15 21:54:44.584 ***</b>
=CRASH REPORT==== 15-Jan-2025::21:54:44.584307 ===
  crasher:
    initial call: peer:init/1
    pid: <0.258627.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-370244-11901@24acc0d94d0f'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2478)
    ancestors: [<0.258601.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1483
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-15 21:54:45.547 ***</b>
=WARNING REPORT==== 15-Jan-2025::21:54:44.545134 ===
Process <0.92.0> on node 'peer_SUITE-shutdown_stop-370276-11901@24acc0d94d0f' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-15 21:54:45.875 ***</b>
=WARNING REPORT==== 15-Jan-2025::21:54:45.875711 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-370308-11901@24acc0d94d0f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-15 21:54:46.087 ***</b>
=WARNING REPORT==== 15-Jan-2025::21:54:46.087733 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-435683-11901@24acc0d94d0f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-01-15 21:55:01.446 ***</b>
=WARNING REPORT==== 15-Jan-2025::21:55:01.446669 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-449829-11901@24acc0d94d0f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-01-15 21:55:01.617 ***</b>
=WARNING REPORT==== 15-Jan-2025::21:55:00.616043 ===
Process <0.89.0> on node 'peer_SUITE-shutdown_stop-449861-11901@24acc0d94d0f' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-01-15 21:55:02.167 ***</b>
=WARNING REPORT==== 15-Jan-2025::21:55:02.167802 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-449893-11901@24acc0d94d0f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-15 21:55:04.335 ***</b>
=ERROR REPORT==== 15-Jan-2025::21:55:04.335215 ===
** Generic server <0.258976.0> terminating 
** Last message in was {'EXIT',<0.258966.0>,
                               {#Ref<0.4100474249.3199205379.181223>,1628628,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-499650-11901",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-499650-11901@24acc0d94d0f',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,50,47,98,105,110,47,101,114,108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-499650-11901",
                                     "-user","peer"],
                                    #Port<0.3130>,undefined,<<>>,running,
                                    {<0.258966.0>,
                                     #Ref<0.4100474249.3206545409.231849>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.4100474249.3199205379.181223>,1628628,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-15 21:55:04.335 ***</b>
=CRASH REPORT==== 15-Jan-2025::21:55:04.335618 ===
  crasher:
    initial call: peer:init/1
    pid: <0.258976.0>
    registered_name: []
    exception exit: {#Ref<0.4100474249.3199205379.181223>,1628628,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.258966.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3130>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 987
    stack_size: 29
    reductions: 4259
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-15 21:55:04.867 ***</b>
=ERROR REPORT==== 15-Jan-2025::21:55:04.867648 ===
** Generic server <0.259006.0> terminating 
** Last message in was {'EXIT',<0.258967.0>,
                           {#Ref<0.4100474249.3206545409.231742>,2161547,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "KYHIYHFNYRMULQFUFHUZ","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@24acc0d94d0f.2025-01-15_21.37.54/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-499618-11901",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@24acc0d94d0f.2025-01-15_21.37.54/make_test_dir.stdlib_test.logs/run.2025-01-15_21.38.30/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-499618-11901",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5.6/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-499618-11901@24acc0d94d0f',
                                    "/usr/local/lib/erlang-26.2.5.6/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-499618-11901",
                                     "-setcookie","KYHIYHFNYRMULQFUFHUZ",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@24acc0d94d0f.2025-01-15_21.37.54/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-499618-11901",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@24acc0d94d0f.2025-01-15_21.37.54/make_test_dir.stdlib_test.logs/run.2025-01-15_21.38.30/log_private/",
                                     "-user","peer"],
                                    #Port<0.3156>,undefined,<<>>,running,
                                    {<0.258967.0>,
                                     #Ref<0.4100474249.3199205380.37340>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.4100474249.3206545409.231742>,2161547,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-15 21:55:04.868 ***</b>
=CRASH REPORT==== 15-Jan-2025::21:55:04.868061 ===
  crasher:
    initial call: peer:init/1
    pid: <0.259006.0>
    registered_name: []
    exception exit: {#Ref<0.4100474249.3206545409.231742>,2161547,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.258967.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3156>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4284
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-15 21:55:05.203 ***</b>
=WARNING REPORT==== 15-Jan-2025::21:55:04.201845 ===
Process <0.89.0> on node 'peer_SUITE-shutdown_stop-499746-11901@24acc0d94d0f' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-15 21:55:05.446 ***</b>
=WARNING REPORT==== 15-Jan-2025::21:55:05.446810 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-499554-11901@24acc0d94d0f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-15 21:55:05.581 ***</b>
=WARNING REPORT==== 15-Jan-2025::21:55:05.581816 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-499586-11901@24acc0d94d0f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-15 21:55:05.787 ***</b>
=WARNING REPORT==== 15-Jan-2025::21:55:05.787753 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-450469-11901@24acc0d94d0f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-15 22:16:41.390 ***</b>
=ERROR REPORT==== 15-Jan-2025::22:16:41.390187 ===
** Generic server <0.335900.0> terminating 
** Last message in was {'EXIT',<0.335897.0>,
                               {#Ref<0.4100474249.3238002689.153538>,351241,
                                [{child1,<0.335906.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.335900.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.335906.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458426],
                            1,0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.4100474249.3238002689.153538>,351241,
    [{child1,<0.335906.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-15 22:16:41.390 ***</b>
=ERROR REPORT==== 15-Jan-2025::22:16:41.390187 ===
** Generic server <0.335899.0> terminating 
** Last message in was {'EXIT',<0.335897.0>,
                               {#Ref<0.4100474249.3238002689.153538>,351241,
                                [{child1,<0.335906.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.335899.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.335905.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458426],
                            1,0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.4100474249.3238002689.153538>,351241,
    [{child1,<0.335906.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-15 22:16:41.390 ***</b>
=CRASH REPORT==== 15-Jan-2025::22:16:41.390626 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.335900.0>
    registered_name: []
    exception exit: {#Ref<0.4100474249.3238002689.153538>,351241,
                     [{child1,<0.335906.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.335897.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1027
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-15 22:16:41.391 ***</b>
=CRASH REPORT==== 15-Jan-2025::22:16:41.390843 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.335899.0>
    registered_name: []
    exception exit: {#Ref<0.4100474249.3238002689.153538>,351241,
                     [{child1,<0.335906.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.335897.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1009
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_bridge_SUITE:simple_global_supervisor/1 2025-01-15 22:17:24.960 ***</b>
=CRASH REPORT==== 15-Jan-2025::22:17:24.960040 ===
  crasher:
    initial call: supervisor_bridge:supervisor_bridge_SUITE/1
    pid: <0.436069.0>
    registered_name: []
    exception exit: killed
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [bridge9212,<0.436064.0>]
    message_queue_len: 0
    messages: []
    links: [<0.436070.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 388
  neighbours:
    neighbour:
      pid: <0.436070.0>
      registered_name: work_supervisor_bridge_SUITE_server
      initial_call: {supervisor_bridge_SUITE,internal_loop_init,1}
      current_function: {supervisor_bridge_SUITE,internal_loop,1}
      ancestors: []
      message_queue_len: 0
      links: [<0.436069.0>]
      trap_exit: false
      status: waiting
      heap_size: 233
      stack_size: 3
      reductions: 9
      current_stacktrace: [{supervisor_bridge_SUITE,internal_loop,1,
                      [{file,"supervisor_bridge_SUITE.erl"},{line,152}]}]

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during zzz_SUITE:lc_graph/1 2025-01-15 22:43:01.189 ***</b>
=WARNING REPORT==== 15-Jan-2025::22:43:01.186211 ===
Process <0.439553.0> on node 'test_server@24acc0d94d0f' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2025 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Wed Jan 15 2025 21:38:30<!--/date--><br />
</div>
</center>
</body>
</html>
